src/pages/Thanks/ThanksPage.vue

```diff
@@ -274,6 +275,9 @@ export default {
 			.filter(item => item.basketItemType === 'loan_reservation')
 			.map(item => item.loan);
 
+		// MARS-194-User metrics A/B Optimizely experiment
+		handleUserMetricsCookies(this.cookieStore, this.loans, this.receipt);
```

Comment:
Could we pass only the loan count and the deposit total to this method instead? Seems like a whole lot of data to pass for 2 very small data points.

Reply:
This is not a blocking suggestion, just a potential optimization to clarify what data we pass and how it's used.